test/e2e/specs/test.js

```diff
@@ -1,12 +1,36 @@
 // For authoring Nightwatch tests, see
 // http://nightwatchjs.org/guide#usage
+var config = require('../../../config')
+var port = process.env.PORT || config.dev.port
+var ui = require('../ui-properties/ui-locators')
+var host = config.dev.host || require('os').hostname() || "localhost"
 
 module.exports = {
   'default e2e tests': function (browser) {
     browser
     .url('http://localhost:8080')
       .waitForElementVisible('#app', 5000)
       .assert.elementPresent('.randPersonBtn')
+      .pause(5000)
+      .end()
+  },
+
+  'starship e2e tests': function (browser) {
+    browser
+    .url('http://' + host + ':'+port)
```

Comment:
A cleaner way to do this would to export the complete url for the test server and then reference that variable in all of the tests (instead of building the url in each of the tests)